ross/tests/test_materials.py

```diff
@@ -1,7 +1,9 @@
 import pytest
 from ross.materials import *
 from numpy.testing import assert_allclose
+import os
 
+os.chdir(os.getcwd()[:os.getcwd().find('tests')])
```

Comment:
This line is used to get to the directory where the __init__ file is, but this will be replaced by the following line:
>>> os.chdir(os.path.dirname(ross.__file__))